validator/sequences/feature_enumeration/sequence.md

```diff
@@ -11,5 +11,6 @@
     * Remove `discovery.generation`
 1. Wait for cleared enumeration generation
 1. Check that no family enumeration
-1. Check that feature enumeration feature is stable
+1. Check that feature enumeration matches metadata
```

Comment:
Should the "match metadata" be a seperate test? I'm thinking about the process here:
1) feature enumeration
2) validate using sequencer that the device supports each feature it says it supports
3) put results into metadata
4) run test again?

Reply:
I'm not sure what you're getting at with your process... the metadata should indicate what features it supports, or you can give it the -a flag if you don't know yet. This test is specifically that the feature discovery enumeration returns the right set of things that you think you're testing against.  Or you run the feature enumeration test, it fails, and then you capture that data. I'm not sure why anything in there would need to be a separate test?

Reply:
I had a misunderstanding which was cleared up in another comment (what the stage was).

So I think this test is validating that the device supports the features the tester knows it supports - i.e. are the results of feature enumeration accurate. The new test could be "does the device support feature enumeration and return a list of features". In any case, this test is fine I think.